safe_transaction_service/analytics/views_v2.py

```diff
@@ -13,10 +13,5 @@ class AnalyticsMultisigTxsByOriginListView(ListAPIView):
     serializer_class = None
 
     def get(self, request, format=None):
-        redis_key = "analytics_transactions_per_safe_app"
-        redis = get_redis()
-        analytic_result = redis.get(redis_key)
-        if analytic_result:
-            return Response(json.loads(analytic_result))
-        else:
-            return Response([])
+        analytics_service = AnalyticsService()
```

Comment:
Add a function that's cached to get the Analytics service, so it's a singleton and you don't initialize the service every time you use it